style.css

```diff
@@ -277,21 +305,58 @@ body {
   flex-direction: column;
   gap: 2.4rem;
   padding-top: 2.4rem;
-  padding-bottom: 12rem;
+  padding-bottom: 2rem;
+}
+
+.open-modal {
+  width: 90%;
+  height: 90vh;
+  background-color: #fff;
+  position: fixed;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  padding: 1.6rem;
+  padding-bottom: 2.4rem;
+  margin: auto;
+  z-index: 5;
+  box-shadow: 0 0 20px rgb(84, 84, 84);
+}
```

Comment:
Awesome work on the desktop version of your popup! I have a couple of friendly suggestions to make it even more professional and aligned with the given [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=Gl1dsCnjASkVGKBM-0):

- [ ] Please review the space between your image and the bottom of the popup, as **it appears slightly different from the template**. 🤓
- [ ] I noticed that the placement of your buttons in the desktop version doesn't match the template. To achieve a closer resemblance, kindly position the **right button in the correct place**. 🙌

  ### Your design:
  ![image](https://github.com/Alibaba2023/My_Portfolio/assets/107952724/22d7ef58-2740-4ed9-9370-3aaf4a2616cb)

  ### Figma design:
  ![image](https://github.com/Alibaba2023/My_Portfolio/assets/107952724/3632d7c0-103e-4240-be65-631e224eb461)

  **_Just a friendly reminder that adhering to the template is a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md). Thanks for considering these suggestions, and keep up the great work!_**
![image](https://github.com/Alibaba2023/My_Portfolio/assets/107952724/c24da992-2083-4760-b22b-7f2bf13e775a)